src/igvfd/schemas/configuration_file.json

```diff
@@ -42,7 +42,26 @@
     "dependentSchemas": {
         "$merge": [
             "file.json#/dependentSchemas"
-        ]
+        ],
+        "content_type": {
+            "comment": "Only configuration files with content_type seqspec are allowed to submit seqspec_of.",
+            "if": {
+                "not": {
+                    "properties": {
+                        "content_type": {
+                            "const": "seqspec"
+                        }
+                    }
+                }
+            },
+            "then": {
+                "not": {
+                    "required": [
+                        "seqspec_of"
+                    ]
+                }
+            }
+        }
```

Comment:
would it be reasonable to add some dependencies between `content_type` and `file_format`?

Reply:
I will restrict seqspec to file_format yaml, but don't think other restrictions are necessary.